administrator/components/com_joomgallery/sql/install.mysql.utf8.sql

```diff
@@ -97,6 +97,13 @@ CREATE TABLE IF NOT EXISTS `#__joomgallery_config` (
   `jg_checkupdate` int(1) NOT NULL,
   `jg_filenamewithjs` int(1) NOT NULL,
   `jg_filenamereplace` text NOT NULL,
+  `jg_replaceimgtitle` text NOT NULL,
```

Comment:
Warum wurde als Datentyp 'text' verwendet? Ein Integer-Typ wäre doch ausreichend. Zudem sollten weiter unten in dieser Datei beim Einfügen eines Datensatzes für die Konfiguration Standardwerte für diese neuen Optionen berücksichtigt werden.